packages/scss/src/components/_grid.scss

```diff
@@ -131,7 +131,7 @@
 
 /* SM - Columns */
 
-@media only screen and (min-width: $sm) {
+@media only screen and (min-width: _get($luTheme, "breakpoints.sm")) {
 	.grid-sm {
 		flex-basis: 0;
```

Comment:
I could create a grid modifier de switch between a standard grid and flex.
Easy to do but quite big to write. --> in my backlog